doc/cephfs/fs-nfs-exports.rst

```diff
@@ -113,6 +144,9 @@ Reset NFS Ganesha Configuration
 
 This removes the user defined configuration.
 
+.. note:: With rook deployment ganesha pods needs to be explicitly restarted
+   for the new config blocks to be effective.
```

Comment:
nit: "With a rook deployment, ganesha pods must be explicitly restarted for the new config blocks to be effective."

Reply:
Done